### DISCUSSION TITLE
How to add horizontal layers overlay?

### DISCUSSION THREAD

**Discussion by aleksandar-b** [xyflow/xyflow#2835] [Q&A]
### Please describe the feature that you want to propose

I want to create something like this, where every layer has an overlay (1st, 2nd, 3rd etc). Is it possible to do that?
<img width="1060" alt="smarscape-2 (2)" src="https://user-images.githubusercontent.com/13149550/217061163-1f480453-8508-44a7-b80e-97efd930a79a.png">

**Comment by moklick** [comment]
You could create custom nodes for the layers that are position below (zIndex option) the other nodes and are not draggable or selectable.
